backend/oasst_backend/api/v1/frontend_users.py

```diff
@@ -6,13 +6,28 @@
 from oasst_backend.api.v1 import utils
 from oasst_backend.models import ApiClient
 from oasst_backend.prompt_repository import PromptRepository
+from oasst_backend.user_repository import UserRepository
 from oasst_shared.schemas import protocol
 from sqlmodel import Session
 from starlette.status import HTTP_204_NO_CONTENT
 
 router = APIRouter()
 
 
+@router.get("/", response_model=list[protocol.User])
+def get_users(
+    max_count: int = Query(10, gt=0, le=20),  # TODO: refine bounds
+    ge: str = None,
+    lt: str = None,
+    auth_method: str = None,
+    api_client: ApiClient = Depends(deps.get_api_client),
+    db: Session = Depends(deps.get_db),
+):
+    pr = UserRepository(db, api_client)
+    users = pr.query_users(limit=max_count, ge=ge, lt=lt, auth_method=auth_method)
+    return [utils.prepare_user(u) for u in users]
```

Comment:
I think that it is better to only import the function that you will use at the end, instead of the whole module.

Reply:
importing modules is actually usually the preferred way, and importing individual members is discouraged.
- the entire module code gets executed either way, but
- with module imports, it's always clear where a function comes from

Reply:
I agree with @yk, besides `utils` was already used in such way in this module before my changes

Reply:
Okay nice! Thanks sorry